src/pmod_als.erl

```diff
@@ -71,12 +72,27 @@ start_link(Slot, Opts) ->
 %% making it close to the human eye (555nm). This implies that
 %% return values will be the highest when the ALS is exposed to
 %% wavelengths of green light with a slight yellow tint.
+-spec read() -> 0..255 | no_return().
 read() ->
     Dev = grisp_devices:default(?MODULE),
     case gen_server:call(Dev#device.pid, {read, Dev#device.slot}) of
         {error, Reason} -> error(Reason);
         Result          -> Result
     end.
+
+%% @doc Returns a the percentage of current ambient light
+%% based on the {@link pmod_als:read} function. The value
+%% is rounded to the closest integer.
+-spec percentage() -> 0..100 | {error, atom()}.
+percentage() ->
+    try read() of
+        Raw when is_number(Raw) ->
+            round((( Raw / 255 ) * 100))
+    catch
```

Comment:
I think you don't need the catch here, better to let it go just like in `read()`. I would just make this function:
```
percentage() ->
    Raw = read(),
    round((Raw / 255) * 100).
```